### DISCUSSION TITLE
Possible to flash the Top Bar icon on copy?

### DISCUSSION THREAD

**Discussion by HubKing** [oae/gnome-shell-pano#154] [Ideas]
By default, it shows a notification pop-up, but that kind of blocks the view. There also is sound notification, but the sound does not seem customisable. I wonder if it would be possible to have a third option to let users know that something has been copied: flashing its Top Bar icon. Something like switching between white on black and black on white a few times.

**Comment by callmehanu** [comment]
Or maybe a tiny animación right?

**Reply by oae** [reply]
This is a nice idea.

**Comment by lkraav** [comment]
It's true: when editing an e-mail, a lot of copy-operations occur. Notification flood is a bit like a punch in the face.

Can I somehow make Pano ignore Primary selection?

**Reply by oae** [reply]
You can disable `Sync Primary` option in the settings

**Reply by lkraav** [reply]
I never touched this toggle post-install, it is disabled by default. Still, pano picks up all selections.

**Comment by oae** [comment]
What do you think of this animation?

[Screencast from 2023-03-12 23-48-39.webm](https://user-images.githubusercontent.com/1043714/224572817-db0f7c4a-3c8b-42fb-9598-0ab7684bbf17.webm)

**Reply by callmehanu** [reply]
Subtle!!!
I have an idea about, let me see if tomorrow I can share it.

**Reply by oae** [reply]
I used the default gnome-shell wiggle animation

**Reply by callmehanu** [reply]
But would be posible change a static icon for an animated one?
